packages/android_alarm_manager/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.2.3
+* Move firebase_auth from a dependency to a dev_dependency
```

Comment:
End line with a period.

Reply:
Thought I was being consistent with the other entries, but I guess just the 0.2.2 entry was the only one missing a period. I've cleaned that up as well.